policies/clair/vulnerabilities-check.rego

```diff
@@ -1,42 +1,41 @@
 package required_checks
 
-violation_critical_vulnerabilities[{"msg": msg, "details":{"name": name, "description": description, "url": url}}] {
+violation_critical_vulnerabilities[{"msg": msg, "vulnerabilities-count": vulns_count, "details":{"name": name, "description": description}}] {
   rpms_with_critical_vulnerabilities := {rpm.Name | rpm := input.data[_].Features[_]; count(rpm.Vulnerabilities) > 0; rpm.Vulnerabilities[_].Severity == "Critical"}
   not count(rpms_with_critical_vulnerabilities) == 0
 
   name := "clair_critical_vulnerabilities"
-  msg = sprintf("Found packages with critical vulnerabilities: %s", [concat(", ", rpms_with_critical_vulnerabilities)])
-  description := "The image musn't contain packages that have critical severity vulnerabilities."
-  url := "https://source.redhat.com/groups/public/product-security/content/product_security_folder/understanding_red_hat_security_vulnerabilities_10pdf"
```

Comment:
Removing the `url` field will be in conflict with the [API contract](https://github.com/redhat-appstudio/book/blob/main/ADR/0013-integration-service-api-contracts.md#detailed-conftest-output-json) for this format of output. If we don't have a good alternative at this time, we can maybe have it as empty for now?

Reply:
Great catch, yep, lets keep it empty for now.